packages/sources/src/components/card/card.models.ts

```diff
@@ -7,4 +7,5 @@ export interface Card<TemplateFnReturnType> {
   content: TemplateFnReturnType;
   interactions?: Button[];
   image?: string;
+  dsoCardClicked?: (e: CustomEvent<void>) => void;
```

Comment:
`CustomEvent<DsoCardClickedEvent>` (ofzoiets)

Reply:
`interface DsoCardClickedEvent` hier en in de `card.interfaces.ts`